programs/rodeos/cloner_plugin.cpp

```diff
@@ -302,7 +302,7 @@ void cloner_plugin::set_program_options(options_description& cli, options_descri
    op("filter-name", bpo::value<std::string>(), "Filter name");
    op("filter-wasm", bpo::value<std::string>(), "Filter wasm");
    op("profile-filter", bpo::bool_switch(), "Enable filter profiling");
-   op("disable-undo-stack", bpo::bool_switch(), "disable undo stack");
+   op("undo-stack", bpo::bool_switch(), "Enable  undo stack");
```

Comment:
The description calls this `enable-undo-stack` which I think would be better.

Reply:
I would suggest not to use bool_swich and use `bpo::value<std::string>()`; in case if we want to change the default back to enabled in the future, we can just change the default value instead of the option name.

Reply:
Done. Thanks.